src/main/kotlin/no/nav/syfo/application/VurderingService.kt

```diff
@@ -38,8 +39,14 @@ class VurderingService(
         )
 
         // TODO: Journalfor vurdering
-        // TODO: Publish new vurdering
 
+        vurderingProducer.publishVurdering(savedVurdering)
+            .map { vurderingRepository.updatePublishedAt(it.uuid) }
+            .onFailure { log.error("Failed to publish vurdering: ${it.message}") }
```

Comment:
Vi kan godt ha en cronjob som sørger for at om noen feiler her, så blir de fortsatt publisert. Tror jeg foretrekker dette alternativet over å ha en cronjob som publiserer hvert 20. sek.
Hva tenker dere? 🙂

Reply:
Vi kan godt prøve det sånn, er ikke så farlig for meg 🤔 Men da vil aldri `journalpostId` være noe annet enn `null` ved publisering, siden den settes i cronjob. Så er kanskje ikke vits i å sende den med i recorden da?

Reply:
Er det i så fall et behov datavarehus har? Trenger vi noen gang i domenet å konsumere der vi spesifikt lurer på om noe er journalført eller ikke?
Det ville jo vært synd om brukerne skulle blitt påvirket av at vi ikke publiserer vurderinger med en gang fordi DVH vil at `journalpostId` skal være satt 🤔
Det er jo pga. brukerne jeg gjerne vil at vi skal publisere med en gang – fordi det egentlig ikke er noen grunn til at de må vente på at en vurdering som akkurat er gjort skal dukke opp i oversikten.

Reply:
Nei, tror ikke datavarehus bryr seg om `journalpostId`. Grunnen til at det har vært i cronjob er nok bare for å separere alt som kan gå galt ut fra det synkrone kallet. 

Det var mer om vi trenger å sende det feltet, men ser vi har gjort det andre steder så det er jo null stress å bare fortsette med det 🤷🏼‍♂️